module-2/backend/week_three.md

```diff
@@ -10,14 +10,35 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+```rails new app_name ```
 2. What do Models generally inherit from in rails?
+```class ModelName < ActiveRecord::Base```...
 3. What do Controllers generally inherit from in a rails project?
+```ApplicationController```
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+```resources :horses, only: [:show]```
 5. What rake task is useful when looking at routes, and what information does it give you?
+```rake routes``` I typed this about 1000 times over the weekend
 6. What is an example of a route helper? When would you use them?
+```horse_path(@horse)``` it is a built in shortcut for links to a route/view
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+
+pass
```

Comment:
`_url` returns the full URL (e.g. `http://www.some-domain.com/horses`, `_path` returns a root-relative path (e.g. `/horses`).